### DISCUSSION TITLE
Video timestamp offset

### DISCUSSION THREAD

**Discussion by CeliaHanna** [biigle/core#565] [Q&A]
The videos I am annotating in BIIGLE are timestamped with UTC time at which they were recorded. This label shows up in QuickTime player on the play bar, but when I upload to Biigle this time is lost and the video player only shows me the duration of the video. Is there a way I can get it to show the UTC time of the videos as shown in Quicktime player (shown below)? Timestamp links to my metadata so is important...

Any advice would be great. Thanks 

<img width="530" alt="Screenshot 2023-04-04 at 12 16 34" src="https://user-images.githubusercontent.com/94908233/229775427-c56e98e6-d286-456f-8033-4bf03863c72f.png">

**Comment by mzur** [comment]
What you describe with the Quicktime player is not yet possible in BIIGLE. There are feature ideas to [show video metadata as an overlay](https://github.com/biigle/core/issues/462) or as [subtitles](https://github.com/biigle/core/issues/472) in the video annotation tool. This would also include timestamps. I think this is also a feature that people from Ifremer are interested in and consider to implement, so there is a chance that this will be possible someday.

I'm curious how your video metadata is stored. Is it stored in an extra file or is this stored directly in the video file somehow? Could you provide an example file that could be used to test an automatic metadata import?

**Reply by CeliaHanna** [reply]
Thanks for your help :) my metadata are on separate excel files, containing coordinates and environmental readings

**Reply by mzur** [reply]
Are the timestamps also in the excel file and QuickTime reads them from there?
